experiments/recfields/sigma_vs_crossover.m

```diff
@@ -62,35 +62,39 @@
 
   %% Analyze raw data for crossover
   numSigmas = size(std_mean, 1);
-  crossover_cpi = nan(numSigmas * numSigmas, 1);
-  sigma_pairs = nan(numSigmas * numSigmas, 2);
-  counter = 1;
+  crossover_cpi = nan(numSigmas, numSigmas);
+  sigma_pairs = nan(numSigmas, numSigmas, 2);
   for ii=1:numSigmas
-    for ij=1:numSigmas
-      %if ij >= ii, continue; end
-
+    for ij=(ii+1):numSigmas
+      % which direction should we be searching for cross-overs?
       ratios = std_mean(ii,:) ./ std_mean(ij,:); % Check when ratio goes over 1
-      if (ratios(1) > 1)  % detect from high to low crossing
-        compareFn = @(idx) ratios(idx) >= 1;
-      else  % detect low to high crossing
-        compareFn = @(idx) ratios(idx) <= 1;
+
+      % -1 means 1.0+ => 0.9-
+      % add 1, as diff has n-1 elements
+      xoverPts = 1 + find(diff(ratios > 1) == -1);
```

Comment:
Take the case: `ratios = [1.01, 0.98, 0.96, 0.95, 0.95]`
This code assumes every element besides 1.01 is a potential crossover point?
And the code will take the one closest to to the middle (so 0.96), correct? Even though the crossover happened somewhere between the first and second index?